openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -124,13 +124,16 @@ def process(self, instance):
 
         # Disable Pan/Zoom.
         pan_zoom = cmds.getAttr("{}.panZoomEnabled".format(preset["camera"]))
-        cmds.setAttr("{}.panZoomEnabled".format(preset["camera"]), False)
+        cmds.setAttr(
+            "{}.panZoomEnabled".format(preset["camera"]),
+            instance.data["panZoom"]
+        )
```

Comment:
I feel like this should be set using a context manager. I suspect currently if you were to press escape during playblasting or any other error occurred during playblasting (e.g. no disk space or whatever) that the scene would be left in a changed state where the panZoomEnabled state of the camera might mismatch what the user had prior to that.

You should be able to use this [`attribute_values`](https://github.com/pypeclub/OpenPype/blob/a74e62a482ae0e8a1ef081cc3a85d3aa1a173258/openpype/hosts/maya/api/lib.py#L443) context manager and do:
```python
remap = {
    "{}.panZoomEnabled".format(preset["camera"]): instance.data["panZoom"]
}

with attribute_values(remap):
    ...
```

Then you also don't need to get the original value here first and revert the original value later manually too.